README.md

```diff
@@ -6,6 +6,8 @@
 ![Cargo format](https://github.com/google/OpenSK/workflows/Cargo%20format/badge.svg?branch=stable)
 [![Coverage Status](https://coveralls.io/repos/github/google/OpenSK/badge.svg?branch=stable)](https://coveralls.io/github/google/OpenSK?branch=stable)
 
+*News:* [PQC paper published!](#Research)
```

Comment:
First reminder will be whenever we merge `develop`. Is there a better mechanism?

Reply:
I'm not sure we'll get a conflict when merging `develop` because of the preceding blank line.

I don't see much options apart from:
- Trying to remember
- Not write anything
- Try to organize the README such that the intended effect of advertising the paper is achieved in a sustainable way (e.g. bubble up the research section, have a specific papers section, use emoticons, etc)

Reply:
Added a date, so news can now be appended and don't necessarily need deprecation.